deploy-chroot/enter-internal.sh

```diff
@@ -3,6 +3,7 @@
 set -euo pipefail
 
 sudo mount -o rw,remount /proc/sys
+sudo mknod --mode='ug=rw,o=' /dev/kvm c 10 "$(grep -w kvm /proc/misc | cut -f 1 -d ' ')"
```

Comment:
How is this going to behave if kvm support happens to be missing?

Reply:
poorly.